src/amoc_api_scenarios_handler.erl

```diff
@@ -17,18 +17,71 @@
 
 -spec trails() -> trails:trails().
 trails() ->
+    RequestBody =
+    #{name => <<"request body">>,
+      in => body,
+      description => <<"request body (as json)">>,
+      required => true,
+      schema =>
+      #{type => <<"object">>,
+        required => [<<"scenario">>, <<"module_source">>],
+        properties =>
+        #{scenario => #{<<"type">> => <<"string">>,
+                        <<"description">> => <<"Name of scenario">>
```

Comment:
Documentation should explain if `scenario` is a module name or a file name ending with `.erl`.